data/foods/condiments.json

```diff
@@ -0,0 +1,99 @@
+{
+  "description": "A list of condiments",
+  "condiments": [
+    "Aioli",
+    "Ajvar",
+    "Amba",
+    "Barbecue sauce",
+    "Biber salçası",
+    "Chili peppers",
+    "Chili sauce",
+    "Chili oil",
+    "Chimichurri",
+    "Chutney",
+    "Cocktail sauce",
+    "Colo-colo",
+    "Crushed red pepper",
+    "Dabu-dabu",
+    "Dip",
+    "Fish paste",
+    "Fish sauce",
+    "Fritessaus",
+    "Fruit preserves",
+    "Fry sauce",
+    "Gochujang",
+    "Guacamole",
+    "Honey dill",
+    "Horseradish",
+    "Hot sauce",
+    "Skhug",
+    "Kachumbari",
+    "Kachumber",
+    "Ketchup",
+    "Fruit ketchup",
+    "Banana ketchup",
+    "Curry ketchup",
+    "Khrenovina",
+    "Kyopolou",
+    "Mayonnaise",
+    "Mignonette sauce",
+    "Milkette",
+    "Monkey gland sauce",
+    "Muhammara",
+    "Mumbo sauce",
+    "Murri",
+    "Mushroom ketchup",
+    "Mustard and mustard-related",
+    "Mostarda",
+    "Mustard oil",
+    "Tewkesbury mustard",
+    "Turun sinappi",
+    "Nutritional yeast",
+    "Olive oil",
+    "Pepper jelly",
+    "Pesto",
+    "Piccalilli",
+    "Pickled fruit",
+    "Pico de gallo",
+    "Mango pickle",
+    "Pickled cucumber",
+    "Pickled onion",
+    "Pickled pepper",
+    "Pinđur",
+    "Popcorn seasoning",
+    "Relish",
+    "Chrain",
+    "Ljutenica",
+    "Remoulade",
+    "Salad cream",
+    "Salad dressing",
+    "Salmoriglio",
+    "Salsa",
+    "Salsa golf",
+    "Salt",
+    "Salt and pepper",
```

Comment:
Shouldn't this be separated? Combination of ingredients/condiments are more of a final product :)